.npmrc

```diff
@@ -1 +1,2 @@
 script-shell=sh
+shamefully-hoist=true
```

Comment:
This was the only way I could get `packages/cdn` to build. Without it, it throws some vite errors related to the node-polyfill.

Setting shamefully hoist makes it easy to use code from a dependency that has not been explicitly added to `package.json`

I'm unhappy with that, but I'll fix the cdn build in a separate PR/task.

Reply:
@enesozturk @svenvoskamp any clues as to why this fails with the vite errors?

Reply:
To be specific, here's the error

![CleanShot 2024-07-18 at 17 41 46@2x](https://github.com/user-attachments/assets/cfbffc17-5c74-4750-849c-3bbf2e81a985)